build/signedbuild.yml

```diff
@@ -233,6 +233,7 @@ jobs:
       vsVersion: 16.0
       platform: '$(BuildPlatform)'
       configuration: release
+      msbuildArgs: '/p:SourceLinkCreate=true'
```

Comment:
I don't see where this property is used in this PR.
I would expect to see a Condition in one of your .csproj or .targets files referencing it.
If there's no Condition, perhaps it's not necessary to include it.

Reply:
This is from the SourceLink docs at https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/symbols?view=azure-devops#set-up-the-build-task. I would suspect that it gets consumed by the SourceLink assemblies, although that's just a theory.

Reply:
@tmat FYI

Oh, I see. Thanks. I'd missed that in the docs. I need to chase down where that comes from because, as far as I can tell, it has no effect on the Microsoft.SourceLink.* packages or the rest of the build. I find a few references to it in Blazor and one in dotnet/orleans, but it's not a standard property recognized by the Microsoft.SourceLink NuGets.

For the Microsoft.SourceLink set of NuGet packages, the user-facing MSBuild property is **EnableSourceLink**. You can use it to override the default behavior. See: https://github.com/dotnet/sourcelink/blob/9dd9f81089f875d2452db9d6c8397a8e0b19d688/src/SourceLink.Common/build/Microsoft.SourceLink.Common.props#L19

Reply:
I don't know why the docs specify this property. Shouldn't be needed.

Reply:
No, but I entered https://github.com/MicrosoftDocs/azure-devops-docs/issues/12436 to get the docs fixed. I see you responded there already. Thanks.